serialization/json_ld/examples/converted_from_spdx_2.json

```
@@ -6,43 +6,48 @@
     }
   ],
   "@graph": [
+    {
+      "@id": "_:creationInfo1",
+      "type": "CreationInfo",
+      "specVersion": "3.0.0",
+      "created": "2010-01-29T18:30:22Z",
+      "createdBy": [
+        "spdx-example:SPDXRef-Actor-ExampleCodeInspect",
+        "spdx-example:SPDXRef-Actor-JaneDoe"
+      ],
+      "createdUsing": [
+        "spdx-example:SPDXRef-Actor-LicenseFind-1.0"
+      ],
+      "profile": [
+        "core",
+        "software",
+        "licensing"
+      ],
+      "dataLicense": "https://spdx.org/licenses/CC0-1.0",
+      "comment": "This is the SPDX-2.3 JSON example converted to SPDX-3.0. As there is currently no closure on how to treat licenses, they are omitted here for now."
+    },
     {
       "type": "Tool",
       "spdxId": "spdx-example:SPDXRef-Actor-LicenseFind-1.0",
-      "name": "LicenseFind-1.0"
+      "name": "LicenseFind-1.0",
+      "creationInfo": {"@id": "_:creationInfo1"}
```

Is there a reason, why you do not use this shorthand?
```suggestion
      "creationInfo":  "_:creationInfo1"
```

---

Thanks for the nudge, I realized that we have to change the context to have `"@type": "@id"` for `creationInfo` in order for this to work.